openapi.yml

```diff
@@ -153,6 +153,62 @@ components:
           type: string
           description: The primary image for a Recommendation.
 
+    LegacyFeedItem:
+      type: object
+      required:
+        - id
+        - title
+        - url
+        - excerpt
+        - domain
+        - image_src
+        - raw_image_src
+      properties:
+        id:
+          type: integer
+        title:
+          type: string
+        url:
+          type: string
+        excerpt:
+          type: string
+        domain:
+          type: string
+        image_src:
+          type: string
+        raw_image_src:
+          type: string
+
+    LegacySettings:
+      type: object
+      properties:
+        spocsPerNewTabs:
+          type: number
+        domainAffinityParameterSets:
+          type: object
+        timeSegments:
+          type: array
+          items:
+            type: object
+            required:
+              - id
+              - startTime
+              - endTime
+              - weightPosition
+            properties:
+              id:
+                type: string
+              startTime:
+                type: integer
+              endTime:
+                type: integer
+              weightPosition:
+                type: number
```

Comment:
I have little context around this but if we mean to represent these types as Typescript types then we'll have to use `number` for all of them 🤔 

Unless this is just for the openapi schema?

Reply:
Yeah, this is an openapi schema, which distinguishes between integers and floating-point numbers.